openhantek/src/hantekprotocol/controlStructs.cpp

```diff
@@ -130,7 +130,7 @@ ControlAcquireHardData::ControlAcquireHardData() : ControlCommand(HantekE::Contr
 }
 
 ControlGetLimits::ControlGetLimits(size_t channels)
-    : ControlCommand(HantekE::ControlCode::VALUE, 1), offsetLimit(new OffsetsPerGainStep[channels]) {
```

Comment:
This command must return more than one byte (offset limits per gain per channel).